airflow/utils/clis.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+Utilities module for cli
+"""
+from __future__ import absolute_import
+import functools
+from datetime import datetime
+import sys
+import os
+
+import airflow.models
+from airflow.utils import action_loggers
+
+
+def action_logging(f):
+    """
+    Decorates function to execute function at the same time submitting action_logging but in CLI context
+    :param f: function instance
+    :return: wrapped function
+    """
+    @functools.wraps(f)
+    def wrapper(*args, **kwargs):
+        metrics = _build_metrics(f.__name__, *args)
+
+        try:
+            result = f(*args, **kwargs)
+        except:
```

Comment:
do we know what errors we are expecting?

Reply:
No, it doesn't and for this function decorator's context, it only needs to capture if there's error or not. One thing missing here is re-raise the exception and will add it.